client/database.go

```diff
@@ -28,16 +26,16 @@ type Tx interface {
 // Database is the interface for interacting with a specific database
 // in TigrisDB.
 type Database interface {
-	// Transact runs the provided TranactionFunc in a transaction. If the
+	// Transact runs the provided TransactionFunc in a transaction. If the
 	// function returns an error then the transaction will be aborted,
-	// otherwise it will be comitted.
+	// otherwise it will be committed.
 	Transact(ctx context.Context, fn TxFunc) (interface{}, error)
 
-	// ApplySchemasFromDirectory reads all the files in the provided
-	// directory and attempts to apply any files with the .json
-	// extension to the database as collection schemas in a single
-	// transaction.
-	ApplySchemasFromDirectory(ctx context.Context, path string) error
+	ApplySchemas(ctx context.Context, dbName string, schemas []*schema2.Schema) error
```

Comment:
rename it to CreateCollections? and is it possible to accept something like CollectionType?